docs/experiment/infra/evaluation-proxy.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Evaluation Proxy
+description: Containerized service to enhance local evaluation running in your infrastructure.
+icon: simple/docker
+---
+
+!!!beta "Evaluation Proxy is in closed beta."
+    Reach out to your customer success manager or email [experiment@amplitude.com](mailto:experiment@amplitude.com) to request access to this feature.
+
+The Evaluation Proxy is a Service to enable, enhance, and optimize [local evaluation](../general/evaluation/local-evaluation.md) running within your infrastructure.
+
+![Architectural diagram showing how the evaluation proxy interacts with your services and Amplitude.](../../../assets/images/experiment/evaluation-proxy.drawio.svg)
+
+:material-check-circle: **Enable local evaluation on unsupported platforms**:<br /> Use remote [Evaluation APIs](../apis/evaluation-api.md) and [SDKs](../index.md#sdks) to run local evaluation in your infrastructure.
+
+:material-check-circle: **Automatically track assignment events for local evaluations**:<br /> Identical assignment events are deduplicated for 24 hours.
+
+:material-check-circle: **Enhance local evaluation with large cohort targeting**:<br /> Targeted cohorts are synced hourly to the Evaluation Proxy and added to the user prior to evaluation.
+
+## Configuration
+
+The evaluation proxy requires keys as environment variables to run. Otherwise, the service crashes on startup.
+
+| Environment Variable | Description |
+| --- | --- |
+| `AMPLITUDE_API_KEY` | The project's [API key](../../guides/amplitude-keys-guide.md#api-key). |
+| `AMPLITUDE_SECRET_KEY` | The project's [secret key](../../guides/amplitude-keys-guide.md#secret-key). |
+| `AMPLITUDE_DEPLOYMENT_KEY` | <span style="max-width:450px;display:inline-block">The key for the deployment to manage. The [deployment key](../../guides/amplitude-keys-guide.md#deployment-key) must exist within the same project as the API and secret key.</span> |
```

Comment:
maybe, call it AMPLITUDE_EXPERIMENT_DEPLOYMENT_KEY to make it more distinguishable here? I followed the link to the keys-guide page and it looks fine there

Reply:
+1